.quickstart/quickstart.yml

```diff
@@ -0,0 +1,17 @@
+database_key: amazon_selling_partner_database
+schema_key: amazon_selling_partner_schema
+
+dbt_versions: ">=1.3.0 <2.0.0"
+
+table_variables:
+  amazon_ads__portfolio_history_enabled:
+    - orders
+    - order_item
+  amazon_selling_partner__using_catalog_module:
+    - item_summary
+
+destination_configurations:
+  databricks:
+    dispatch:
+      - macro_namespace: dbt_utils
+        search_order: [ 'spark_utils', 'dbt_utils' ]
```

Comment:
We aren't actually bringing in spark-utils. Is this a necessary config?

Reply:
I'm honestly unsure -- it seems like we include this in every other package's quickstart.yml, so I'm hesitant to remove it if it's not causing issues.

Reply:
If we aren't using the spark utils package then we don't need this specialized config.